server/app/api/controllers/users.js

```diff
@@ -6,10 +6,32 @@ module.exports = {
   create: function(req, res, next) {
     userModel.create(
       {
-        userName: req.body.userName,
-        email: req.body.email,
-        password: req.body.password,
-        userRole: req.body.userRole
+          employee_id : req.body.employee_id ,
+          email: req.body.email ,
```

Comment:
what will happen if field is not present in request body

Reply:
Validation is displayed for the mandatory field.